src/init/features/dataconnect/index.ts

```diff
@@ -19,7 +19,7 @@ import { Schema, Service, File, Platform } from "../../../dataconnect/types";
 import { parseCloudSQLInstanceName, parseServiceName } from "../../../dataconnect/names";
 import { logger } from "../../../logger";
 import { readTemplateSync } from "../../../templates";
-import { logBullet } from "../../../utils";
+import { logBullet, envOverride } from '../../../utils';
```

Comment:
nit: did these get changed from `"` to `'` intentionally?

Reply:
Nope, no clue why my auto import did that... fixing for cosnistency